platform-hub-web/src/app/help/faq/faq-entries.html

```diff
@@ -53,82 +87,105 @@
+              </li>
+            </ul>
           </li>
           <li>
             You can <a ui-sref="announcements.global">view global announcements about the platform</a>
           </li>
           <li>
-            You can view information and resources about the various platform <em>themes</em> from the sidebar or <a ui-sref="home">homepage</a>
+            You can view information and resources about the various <em>platform themes</em> from the sidebar or <a ui-sref="home">homepage</a>
           </li>
           <li ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
-            You can <a ui-sref="projects.list">view projects and team members</a>
+            You can <a ui-sref="projects.list">view projects and their team members</a>
+
+            <ul>
+              <li>
+                If you are a member of a project, you can also view all of it's services and associated resources (e.g. Kubernetes robot tokens for a particular service within that project)
+              </li>
+            </ul>
           </li>
           <li>
             You can <a ui-sref="help.support.requests.overview">make a support request</a>
           </li>
         </ul>
 
         <p ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
-          If you are made a project admin, you can:
+          If you are made a <strong>project admin</strong> you can:
         </p>
         <ul ng-if="$ctrl.FeatureFlags.isEnabled($ctrl.featureFlagKeys.projects)">
           <li>
-            Add someone to your project team (on the project page)
+            Add someone to your project team (on the main project page).
           </li>
           <li>
-            Remove a team member from your project (using the <em>Actions</em> button next to a team member, on the project page)
+            Remove a team member from your project (using the <em>Actions</em> button next to a team member, on the project page).
           </li>
           <li>
-            Onboard and offboard team members on GitHub (using the <em>Actions</em> button next to a team member, on the project page). <strong>Note:</strong> you can only do this for users that have connected up their GitHub identity within the hub.
+            Onboard and offboard team members on Github (using the <em>Actions</em> button next to a team member, on the project page). <strong>Note:</strong> you can only do this for users that have connected up their Github identity within the hub.
+          </li>
+          <li>
+            Manage the <em>services</em> that are being developed in your project (via the Services tab on your project page).
+
+            <ul>
+              <li>
+                Project services are a way of grouping and breaking up your project into smaller sub-projects / deliverables / components / apps etc. Most resources provided by the platform are scoped to, and associated with, a particular service in your project (e.g. Kubernetes robot tokens).
```

Comment:
In this area I think it would be helpful to give an example structure of the below and their associations:
- Project
- Service
- Namespaces

Maybe using ACP Team as an example, with a service Artifactory split across NotProd & Prod (fitting into the above).

Reply:
I think this is one those things that's very hard (and fairly complex) to explain in text, and more of a "you have to see and experience it to get it" – I wanted to give the bare minimum info here and then have users click around and see it for themselves. So i'm hesitant to add anymore at this stage – thinking we should first get some feedback from users and add more if necessary (and if we can come up with a really compelling example).